admin/make-release.zsh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env zsh
```

Comment:
Just thinking out loud. We should probably stay with `sh` as other admin scripts do. Anyway, it's just a set of simple tasks for which `zsh` does not look to be a requirement.

Reply:
pushd and popd are the reason why I chose zsh. I switched back to /bin/sh though.